rollup.dev.config.js

```diff
@@ -0,0 +1,12 @@
+import typescript from "@rollup/plugin-typescript";
+import autoExternal from "rollup-plugin-auto-external";
+
+export default {
+  input: "src/dev/create-extension.ts",
+  output: {
+    dir: "create-extension-cli",
```

Comment:
I think it would be good if it's clear that dir is generated. Maybe change dir to `generated/create-extension-cli` ?  Or `src/dev/scripts/create-extension-cli` or something like that?

Reply:
Ohh actually removed that dir. Now exporting the output in `dist` dir (which is already `.gitignored`) 

Added a new `script` in `package.json` :  `"build:dev": "yarn build && rollup -c src/dev/rollup.config.js",`


So when people clones this repo to create new extension they need to run `yarn build:dev` which will build our main cli and this create-extension cli  (We should document this later) 


Also updated https://github.com/scaffold-eth/create-eth/pull/73#issue-2385945604 with step 1.5